### ISSUE TITLE
ER-TFTM0784-2

### ISSUE THREAD

**Issue by Eddiiie** [xlatb/ra8876#11]
Hello,

Thank you for this library.   Is there any chance you can add support for the ER-TFTM0784-2 display?  
1280x400 resolution, RA8876, same pinouts, except the touch screen pins..  FONT and FLASH chips.

I need this display for a project but I can't get it to work.  It just needs to display 5 - 6 screens quickly, not seconds.
Is this even possible?

They (buy display) do not offer a library for Arduino.   No problem.  I'll get the 8051 demo board, take the sample code and adapt it.
hahaha  Not so easy.  At least not for me.  I can get the code to say "RA8876 found!" that's it.     RA offers some good Arduino libraries.  Unfortunately, I can't them to work with this display either.

Pulling my hair out.   I am confident my wiring is OK.   Sketches detect the RA8876 chip, and run as expected, but nothing shows up.  Like the backlight isn't on..... I see the display flicker a little when a sketch is uploaded or power is applied/removed. 

I have 2 setups; can I send you one for troubleshooting?

**Comment by Eddiiie** [comment]
I think it would have worked if I tied the backlight to VCC.

**Comment by Eddiiie** [comment]
...

**Comment by u20p17** [comment]
@Eddiiie : did you make it work?

**Comment by Eddiiie** [comment]
Hi, thank you for checking in.     I moved my project to STM32 and CubeIDE.  
The project required 9 bit UART and multi-processor mode which is not really possible in Arduino.
Glad I made the move.  The STM32 has so much more .. everything.  RAM, FLASH, Mhz, 

This display is odd to me.  It seems it is really meant to work in portrait mode (400x1280), but I need it in Landscape mode (1280x400).  This has caused some strange problems.  The text has to be upside down, so I rotate display.  Fine.  
PIP mode, can't use it with text because the text shows backward.   Embedding graphics in the code, we need to flip the graphic to portrait mode and flip again another way..

It lacks basic functionality that I would expect for text..  There is no, center, left, right, reverse, flash, etc functions. 
We have to write all those ourselves.

RAIO is non-responsive.   The buydisplay.com people are actually pretty responsive and helpful.
